spec/support/factory_girl.rb

```diff
@@ -7,4 +7,23 @@
     password_confirmation 'password'
   end
 
+  factory :bar do
+    name "Beacon Hill Pub"
+    address "123 Harrison Ave"
+    city "Boston"
+    state "MA"
+    zip "02156"
+
+    factory :bar2 do
+      name "Drink"
+      address "1234 Quenchtown Lane"
+    end
+
+    factory :bar3 do
+      name "Mayo's Frozen Bar"
+      address "22 Popsicle Lane"
+    end
+
+  end
```

Comment:
I would just create a sequence rather than three different bars:

```ruby
factor :bar do
  sequence(:name) { |n| "Bar Number #{n}" }
  sequence(:address) { |n| "123#{n} Mass Ave." }
  .
  .
  .
end
```